src/ui/public/agg_table/agg_table.js

```diff
@@ -93,6 +95,35 @@ uiModules
             formattedColumn.class = 'visualize-table-right';
           }
 
+          let isFirstValueNumeric =
+            table.rows && table.rows[0] && table.rows[0][i] && _.isNumber(table.rows[0][i].value);
+
+          if ((field && field.type === 'number') || isFirstValueNumeric) {
+            function sum() {
```

Comment:
I would really like this to be more specific. If you only want to use it with table rows, maybe make it a `const`, otherwise this can be a function, but can we make more functional in the sense that it takes a param, and returns transformed data from said param. So something like this;
```javascript
function sum(tableRows) {
return _.reduce(tableRows, function (prev, curr) {return prev + curr[i].value; }, 0);
```